### PR TITLE
Update send_transactions_on_axon_via_metaMask.md

### PR DESCRIPTION
**Pull request by sunchengzhu**

### CODE REVIEW

File: docs/getting-started/for-dapp-devs/send_transactions_on_axon_via_metaMask.md [hunk 1]
```diff
@@ -16,14 +16,16 @@ This guide provides instructions for sending transactions on Axon via MetaMask a
 
 ### 1.1 Local Setup
 
-[Download Axon](https://github.com/axonweb3/axon), find <b>Install</b> in <b>README</b>, and run the following commands to start the node:
+[Download Axon](https://github.com/axonweb3/axon), find <b>Compile</b> in <b>README</b>, and run the following commands to start the node:
 
-`cargo run --release -- -c devtools/chain/config.toml -g devtools/chain/genesis_single_node.json`
+```bash
+target/release/axon run --config devtools/chain/config.toml
```
Comment (Flouse) [inline docs/getting-started/for-dapp-devs/send_transactions_on_axon_via_metaMask.md:22]: This command is not enough to start a Axon chain, missing `axon init`.

Comment (Flouse) [reply docs/getting-started/for-dapp-devs/send_transactions_on_axon_via_metaMask.md:22]: suggestion:
reuse the quick_start doc:

https://github.com/sunchengzhu/axon-docs/blob/313d023c11cfdfec12d6c03483c31aee70e01c49/docs/getting-started/for-dapp-devs/deploy_solidity.md?plain=1#L14

**Review by Flouse — CHANGES_REQUESTED** [review]

File: docs/getting-started/for-dapp-devs/send_transactions_on_axon_via_metaMask.md [hunk 2]
```diff
@@ -67,13 +73,13 @@ Once you have filled out all the items above, click <b>Save</b> and you will be
...
 
-For now, it is `0x37aa0f893d05914a4def0460c0a984d3611546cfb26924d7a7ca6e0db9950a2d`.
+For example, the private key `0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80`, which corresponds to the address `0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266`, is derived from the mnemonic phrase `test test test test test test test test test test test junk` for the first account. This mnemonic phrase is consistent with [the one used for the initialization of the Hardhat Network](https://hardhat.org/hardhat-network/docs/reference#initial-state).
 
-The account and the balance will be displayed once the account is added. The Genesis account holds 1000,000,000,000,00 AXON configured in `devtools/chain/config.toml`.
+The account and the balance will be displayed once the account is added. This Genesis account holds 100,000,000,000,000 AXON, as configured in [`devtools/chain/specs/single_node/chain-spec.toml`](https://github.com/axonweb3/axon/blob/6fe5777e0b4a9b994dc84a56a00005745fd05085/devtools/chain/specs/single_node/chain-spec.toml#L20).
```
Comment (Flouse) [inline docs/getting-started/for-dapp-devs/send_transactions_on_axon_via_metaMask.md:80]:
```suggestion
The account and the balance will be displayed once the account is added. This Genesis account holds 100,000,000,000,000 Axon Token (AT), as configured in [`devtools/chain/specs/single_node/chain-spec.toml`](https://github.com/axonweb3/axon/blob/6fe5777e0b4a9b994dc84a56a00005745fd05085/devtools/chain/specs/single_node/chain-spec.toml#L20).
```

File: docs/getting-started/for-dapp-devs/send_transactions_on_axon_via_metaMask.md (hunk 1 above)
Comment (sunchengzhu) [reply docs/getting-started/for-dapp-devs/send_transactions_on_axon_via_metaMask.md]: > suggestion: reuse the quick_start doc:
> 
> https://github.com/sunchengzhu/axon-docs/blob/313d023c11cfdfec12d6c03483c31aee70e01c49/docs/getting-started/for-dapp-devs/deploy_solidity.md?plain=1#L14

Modified in https://github.com/axonweb3/axon-docs/pull/205/commits/be24182d8cacf543bec5a5199a62bb12a9b54624

**Review by Flouse — APPROVED** [review]

**Review by linnnsss — APPROVED** [review]

**Pull request merged**